Program.cs

```diff
@@ -1,15 +1,17 @@
 ﻿using System;
 using System.Linq;
-for (; ; ) {
-   Console.Write ("Enter the word : ");
-   string word = Console.ReadLine ().ToLower ();
-   char[] input = word.ToArray ();
-   Array.Sort (input);
-   string sortedInput = "";
-   foreach (char c in input) {
-      sortedInput += c;
-   }
-   string result = (word == sortedInput) ? "It is an Abecedarian word." : "It is NOT an Abecedarian word.";
-   Console.WriteLine (result);
-   Console.WriteLine ("Longest Abecedarian word : " + sortedInput);
+string[] words = { "apple", "effort", "cherry", "glory", "fig", "grape", "aegilops", "adopt", "mother", "office" };
+string longestAbecedarianWord = FindLongestAbecedarianWord (words);
+var res = string.IsNullOrEmpty (longestAbecedarianWord) ? $"No abecedarian word found in the array." : $"Longest Abecedarian Word:{longestAbecedarianWord}";
+Console.WriteLine (res);   
+
+bool IsAbecedarian (string word) {

```

Comment:
You can write "IsAbecedarian" function like this,
`bool IsAbecedarian (string word) => word == new string (word.OrderBy (c => c).ToArray ());`